packages/cli/src/models/files/NetworkFile.ts

```diff
@@ -88,7 +88,7 @@ export default class NetworkFile {
   public data: NetworkFileData;
 
   public static getManifestVersion(network: string): string | null {
-    const file = fs.parseJsonIfExists(`zos.${network}.json`);
+    const file = fs.existsSync(`zos.${network}.json`) ? fs.readJsonSync(`zos.${network}.json`) : null;
```

Comment:
I agree, I regret going with `{ throws: false }`, though I think it still makes sense for reading the `package.json`. The effect is that it will ignore any sort of error in reading the file, including JSON syntax errors. It made sense to me to ignore any errors when reading the `package.json` just to read a default version number.